### PR TITLE
7782 - Reset org name on business type change

### PR DESCRIPTION
**Pull request by karthik-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/7782
*Description of changes:*

- Reset org name while switching business type

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by sumesh-aot — APPROVED** [review]

File: auth-web/src/components/auth/common/AccountBusinessType.vue [hunk 1]
```diff
@@ -8,7 +14,7 @@
             row
             v-model="isBusinessAccount"
             ref="isBusinessAccount"
-            @change="onOrgBusinessTypeChange"
+            @mouseup="onOrgBusinessTypeChange(true)"
```
Comment (shabeeb-aot) [inline auth-web/src/components/auth/common/AccountBusinessType.vue:17]: mouse-up may not work if some one is using tabs?

File: auth-web/src/components/auth/common/AccountBusinessType.vue [hunk 2]
```diff
@@ -245,10 +256,14 @@ export default class AccountBusinessType extends Vue {
     }
 
     // emit the update value to the parent
-    await this.onOrgBusinessTypeChange()
+    await this.onOrgBusinessTypeChange(false)
   }
 
-  async onOrgBusinessTypeChange () {
+  async onOrgBusinessTypeChange (clearOrgName: boolean) {
```
Comment (shabeeb-aot) [inline auth-web/src/components/auth/common/AccountBusinessType.vue:262]: clearOrgName default value will be good

File: auth-web/src/components/auth/common/AccountBusinessType.vue (hunk 1 above)
Comment (karthik-aot) [reply auth-web/src/components/auth/common/AccountBusinessType.vue]: Changed to @change in v-radio buttons. It does the job.

File: auth-web/src/components/auth/common/AccountBusinessType.vue (hunk 2 above)
Comment (karthik-aot) [reply auth-web/src/components/auth/common/AccountBusinessType.vue]: Thanks for this suggestion. Refactored

**Review by shabeeb-aot — APPROVED** [review]

**Pull request merged**